Formula/rom-tools.rb

```diff
@@ -30,6 +30,13 @@ class RomTools < Formula
   uses_from_macos "expat"
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "gcc" # for C++17
```

Comment:
`qt@5`, `sdl2_ttf` (I can't check `pluseaudio`) should be build dependencies as it will build `mame` binary.

Reply:
I'll check the linkage once a bottle is available. Mainly based on other Linux distros like Arch https://github.com/archlinux/svntogit-community/blob/packages/mame/trunk/PKGBUILD#L50